presto-main/src/test/java/io/prestosql/sql/query/TestDistinctAggregations.java

```diff
@@ -272,4 +272,14 @@ public void testMixedDistinctWithFilter()
                         "FROM (VALUES (1, 3), (2, 4), (2, 4), (4, 5)) t (x, y)",
                 "VALUES (BIGINT '0', CAST(NULL AS BIGINT))");
     }
+
+    @Test
+    public void testUuidDistinct()
```

Comment:
Can we add a similar test for `testIPAddressDistinct()` as it faces similar issue

Reply:
@Praveen2112 I wish none of these was necessary -- https://github.com/prestosql/presto/pull/1180#issuecomment-515005779

Reply:
> Can we add a similar test for `testIPAddressDistinct()` as it faces similar issue

@Praveen2112  There's no issue with `distinct` and `group by` for `IPAddress`
This one works: `SELECT DISTINCT ipaddress_col FROM (VALUES (IPADDRESS '10.0.0.1'),(IPADDRESS '10.0.0.1')) AS t (ipaddress_col)`